app/controllers/schools/dashboard_controller.rb

```diff
@@ -1,8 +1,7 @@
 # frozen_string_literal: true
 
 class Schools::DashboardController < Schools::BaseController
-  before_action :set_school_cohorts, only: :show, unless: -> { FeatureFlag.active?(:multiple_cohorts) }
-  before_action :set_multi_cohorts, only: :show, if: -> { FeatureFlag.active?(:multiple_cohorts) }
+  before_action :set_multi_cohorts, only: :show
```

Comment:
It doesn't feel like `set_multi_cohorts` is a good name for this method any more, perhaps just `set_school_cohorts`?

Perhaps the check to ensure some exist should be split out too, but it's not as important as the method name.

Reply:
Renamed and split